src/routes/donations.js

```diff
@@ -61,8 +62,10 @@ router.post('/upload', (req, res) => {
         }, Promise.resolve([]))
       })
       .then(results => {
-        const deduped = _dedupeDonors(results)
-        res.status(200).send(deduped)
+        const deduped = () => {
+          return [_groupDonors(results), summary(results)]
```

Comment:
Rather than send back an array, we can send back an object with keys that reflect what is what:

```js
{
  data: _groupDonors(results),
  summary: summary(results)
}
```

Reply:
okay done , committed and push !